test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -35,6 +37,39 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
     }
+    
+    @Test
+    public void utils_isAnyNull_returnsTrue() {
+        
+        //only one null object
+        assertTrue(isAnyNull((Object) null));
```

Comment:
This LO requires you to create an isAnyNull() method instead of using the method inside utils. You may want to check the other's PR :)